### DISCUSSION TITLE
安卓APP无法登陆问题

### DISCUSSION THREAD

**Discussion by cqbcqb** [getlantern/lantern-forum-cn#224] [General]
安卓的APP大多数时间无法连接到服务器，无法使用

**Comment by MaxMKMax** [comment]
@cqbcqb 
您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。

**Reply by cqbcqb** [reply]
70125942，70149521

**Reply by MaxMKMax** [reply]
@cqbcqb 好的. 非常谢谢您帮忙提交应用内报告. 我可以拿您的问题来上报给科技团队了哦!

**Reply by cqbcqb** [reply]
我重新下载了APP，但是APP是全英文的，语言切换不了，而且在APP中无法登陆专业版

**Reply by MaxMKMax** [reply]
@cqbcqb 好的。对带来的麻烦表示抱歉。我会向团队反馈语言和登录问题。您能告诉我您使用的是哪些设备吗？例如，桌面window 10、macOS android、ios？

您在哪里下载该应用程序？

**Reply by cqbcqb** [reply]
安卓系统，在官网下载的

**Reply by MaxMKMax** [reply]
@cqbcqb 谢谢你让我知道。我会将问题告知团队以进行改进!
